### ISSUE TITLE
java.lang.RuntimeException: Unable to start activity Bad magic number for Bundle: 0x6f0063

### ISSUE THREAD

**Issue by javajack** [sockeqwe/mosby#24]
http://crashes.to/s/7ed95959eaa/details

**Comment by sockeqwe** [comment]
Really strange :)

I guess it's related to #23 and will be fixed with mosby 1.1 as well.

**Comment by sockeqwe** [comment]
@javajack Just wanted to ask if you had the chance to update your app to `mosby 1.1.0` and if yes, you still get some related crash reports or is it fixed?

Btw. I have installed your app. The vertical scrolling is pretty nice, is it a kind of vertical view pager? Well done!

**Comment by javajack** [comment]
@sockeqwe 
Yes I have upgraded and pushed the latest Mosby( TimesBuzz version 1.1.1 ).

Will share the exception logs, if any ?
Since the app is just launched for a week or two the user base is tiny and they are on old versions.
It may take some time as news users on board with latest version and old users upgrade.

"The vertical scrolling is pretty nice.." That was the vision.
I was handicapped with overall Android architecture and development.
I am not a good programmer. 

I just hustled a bit and leveraged what you and other people had to offer ;) 
If you want could look at my code, I'll be glad to add you in my private bitbucket repo.

Just ping me your bitbucket username or email id :)
